recipes/cuda-python/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[1] %}
+{% set patch_version = version.split(".")[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true  # [py<37 or osx]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - setuptools
+    - cython
+    - pycparser
+    - autopxd2
+    - pyclibrary
+    - python
+    - pip
+  run:
+    - python
+    - cudatoolkit >={{ major_version ~ ".0.0" }}
+  run_constrained:
+    - __cuda >={{ major_version ~ ".0.0" }}
+    - __glibc >=2.17  # [linux]
+
+test:
+  imports:
+    - cuda
+    - cuda.cuda
+    - cuda.cudart
+
+about:
+  home: https://github.com/NVIDIA/cuda-python
+  license: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+  license_file: LICENSE
+  summary: 'CUDA Python Low-level Bindings'
+  description: |
+    CUDA Python provides a standard set of low-level interfaces,
+    providing full coverage of and access to the CUDA host APIs from Python.
+  doc_url: https://nvidia.github.io/cuda-python/
+  dev_url: https://github.com/NVIDIA/cuda-python
+
+extra:
+  recipe-maintainers:
+    - m3vaz
+    - jakirkham
+    - mmccarty
```

Sorry I am late to the party! 😂
```suggestion
    - mmccarty
    - leofang
```